### ISSUE TITLE
Add confirmation dialog before script execution

### ISSUE THREAD

**Issue by mtrezza** [parse-community/parse-dashboard#2474]
### New Feature / Enhancement Checklist

- [x] I am not disclosing a [vulnerability](https://github.com/parse-community/parse-server/blob/master/SECURITY.md).
- [x] I am not just asking a [question](https://github.com/parse-community/.github/blob/master/SUPPORT.md).
- [x] I have searched through [existing issues](https://github.com/parse-community/parse-dashboard/issues?q=is%3Aissue).

### Current Limitation

A script is executed as soon as it is selected from the context menu. For sensitive operations it should be possible to show a confirmation dialog before the script is executed.

### Feature / Enhancement Description

Add a config option to the script that shows a confirmation dialog, which allows the user to either "Cancel" (default / highlight button) or "Continue". By default, no dialog should be displayed.

The dialog could look like this:

<img width="552" alt="Screenshot 2023-06-23 at 20 59 02" src="https://github.com/parse-community/parse-dashboard/assets/5673677/5d73a3f1-7328-4dbd-949d-0a2226ee47fe">

In addition, the confirmation dialog should have 2 levels of severity, as is currently implemented throughout the dashboard UI. A "critical" level (red style) and an "info" level (blue style), like this:

<img width="557" alt="image" src="https://github.com/parse-community/parse-dashboard/assets/5673677/8612ef2e-3a71-4525-bff6-f1c97a190a25">

### Example Use Case

```json
"apps": [
  {
    "scripts": [
      {
        "title": "Delete Account",
        "classes": ["_User"],
        "cloudCodeFunction": "deleteAccount",
        "showConfirmationDialog": true,
        "confirmationDialogStyle": "critical" // or "info" for blue style
      }
    ]
  }
]
```

### Alternatives / Workarounds

None

**Comment by patelmilanun** [comment]
do we want the dialog text as well?

**Comment by mtrezza** [comment]
You mean a customizable dialog text? That would be a nice addition, but it would need to support some build-in variables, like `{{scriptName}}`, `{{className}}`, `{{objectId}}`, in order to compose a specific message text, like in the example above. If no text is set, it could just display the text in the example above by default:

> Do you want to run the script "Delete User" on "_User" object "Abc123Xy"?

**Comment by parseplatformorg** [comment]
🎉 This change has been released in version [5.2.0-alpha.24](https://github.com/parse-community/parse-dashboard/releases/tag/5.2.0-alpha.24)

**Comment by parseplatformorg** [comment]
🎉 This change has been released in version [5.3.0-beta.1](https://github.com/parse-community/parse-dashboard/releases/tag/5.3.0-beta.1)

**Comment by parseplatformorg** [comment]
🎉 This change has been released in version [5.3.0-alpha.1](https://github.com/parse-community/parse-dashboard/releases/tag/5.3.0-alpha.1)

**Comment by parseplatformorg** [comment]
🎉 This change has been released in version [5.3.0](https://github.com/parse-community/parse-dashboard/releases/tag/5.3.0)
